libqtile/backend/wayland/cffi/Makefile

```diff
@@ -0,0 +1,29 @@
+# Paths
+QW_PATH      := ../qw
+QW_PROTO_OUT := ../qw/proto
+
+# Compiler/linker
+CC := gcc
+CFLAGS := -O2 -fPIC \
+          -DWLR_USE_UNSTABLE \
+          -I$(QW_PATH) \
+          -I$(QW_PROTO_OUT) \
+          $(shell pkg-config --cflags wlroots-0.19 wayland-server cairo pixman-1 libdrm)
+
+LDFLAGS := -shared $(shell pkg-config --libs wlroots-0.19 wayland-server cairo pixman-1 libdrm)
```

Comment:
LDLIBS right? not LDFLAGS? Then you don't need to pass it explicitly below (same with CFLAGS)